lms/djangoapps/lti_provider/views.py

```diff
@@ -28,7 +31,7 @@
 REQUIRED_PARAMETERS = [
     'roles', 'context_id', 'oauth_version', 'oauth_consumer_key',
     'oauth_signature', 'oauth_signature_method', 'oauth_timestamp',
-    'oauth_nonce', 'user_id'
+    'oauth_nonce', 'user_id', 'resource_link_id', 'lti_version', 'lti_message_type'
```

Comment:
I assume that validating these were passed is a requirement of the certification test? If so, we should keep the (maybe a clear exception message in case it does fail in the field we can quickly triage). If not required by the tests, I agree we shouldn't require them either.